client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/PrescriptionLineEditForm.tsx

```diff
@@ -240,7 +240,7 @@ export const PrescriptionLineEditForm: React.FC<
               autoFocus
               value={issueQuantity}
               onChange={handleIssueQuantityChange}
-              min={1}
+              min={0}
```

Comment:
To be consistent with the above. We could make it default back to `1` when undefined, but then there's not point having the warning message.